cardano-cli/src/Cardano/CLI/EraBased/Options/Address.hs

```diff
@@ -14,22 +14,27 @@ import           Cardano.CLI.EraBased.Commands.Address
 import           Cardano.CLI.EraBased.Options.Common
 
 import           Data.Foldable
+import           Data.Maybe
 import           Options.Applicative hiding (help, str)
 import qualified Options.Applicative as Opt
 
 pAddressCmds :: CardanoEra era -> EnvCli -> Parser (AddressCmds era)
 pAddressCmds _ envCli =
-  asum
-    [ subParser "key-gen"
+  asum $ catMaybes
```

Comment:
I don't think this is necessary as these commands will be available in every era unless e.g the crypto changes.